tests/pageobjects/installed-snaps-page.js

```diff
@@ -59,7 +59,7 @@ var installedSnapsPage = Object.create(basepage, {
                 });
             });
             acPage.submit_token(valid_token);
-            loginpage = browser.element('h2=Installed snaps');
+            loginpage = browser.element(".p-card h3.js-snap-title");
```

Comment:
I would agree with elopio's comment. For every testable element, there should be id or name as those are most efficient locater strategies, however most of the elements on snapweb missing the ids and names.